content/working_with/console/customization/overrides.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bt_wiki
+title: Labels Overrides
+category: Console
+draft: false
+weight: 180
+---
+
+You can override a number of texts and labels used in {{< param cfy_console_name >}} by supplying a file containing replacements for default texts or labels.
+
+To do that:
+
+1. Create `/opt/cloudify-stage/dist/userData/overrides.json` file.
+1. Go to https://github.com/cloudify-cosmo/cloudify-stage/blob/master/app/translations/en.json, choose configurations parts to modify (you can omit all unmodified parts to use default values) and save them to the created file.
```

Comment:
As labels can change over time, to be more precise we can also write about an option to switch to specific release (by selecting a branch `${version}-build`).